### ISSUE TITLE
Argument expected but is not required

### ISSUE THREAD

**Issue by NekoTony** [TheTrain2000/async2rewrite#11]
```
root@ubuntu-512mb-nyc3-01:~/modules# python3.5 -m async2rewrite shop.py --suffix  .edited.py
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/__main__.py", line 18, in <module>
    converted = from_file(*results.paths, interactive=results.interactive)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/main.py", line 76, in from_file
    res[path] = process_file(path, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/main.py", line 62, in process_file
    return get_result(f.read(), **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/main.py", line 42, in get_result
    new_ast = DiscordTransformer(interactive=interactive).generic_visit(expr_ast)
  File "/usr/lib/python3.5/ast.py", line 300, in generic_visit
    value = self.visit(value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/lib/python3.5/ast.py", line 300, in generic_visit
    value = self.visit(value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/transformers.py", line 131, in visit_AsyncFunctionDef
    self.generic_visit(node)
  File "/usr/lib/python3.5/ast.py", line 300, in generic_visit
    value = self.visit(value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/lib/python3.5/ast.py", line 300, in generic_visit
    value = self.visit(value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/lib/python3.5/ast.py", line 300, in generic_visit
    value = self.visit(value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/lib/python3.5/ast.py", line 300, in generic_visit
    value = self.visit(value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/lib/python3.5/ast.py", line 300, in generic_visit
    value = self.visit(value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/transformers.py", line 51, in visit_Expr
    self.generic_visit(node)
  File "/usr/lib/python3.5/ast.py", line 309, in generic_visit
    new_node = self.visit(old_value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/transformers.py", line 126, in visit_Await
    self.generic_visit(node)
  File "/usr/lib/python3.5/ast.py", line 309, in generic_visit
    new_node = self.visit(old_value)
  File "/usr/lib/python3.5/ast.py", line 245, in visit
    return visitor(node)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/transformers.py", line 68, in visit_Call
    node = self.stateful_edit_message(node)
  File "/usr/local/lib/python3.5/dist-packages/async2rewrite/transformers.py", line 557, in stateful_edit_message
    content = call.args[1]
IndexError: list index out of range

```

**Comment by tylergibbs2** [comment]
Please send the code that you tried to convert, this looks like a known bug.

**Comment by NekoTony** [comment]
https://hastebin.com/orevapoyed.py

**Comment by tylergibbs2** [comment]
This looks like a bug similar to #6, but not exactly the same. As a temporary work around, you can change
`await self.bot.edit_message(reaction.message, embed=embed)`
to
`await self.bot.edit_message(reaction.message, ' ', embed=embed)`

**Comment by tylergibbs2** [comment]
This has been solved in the latest release (0.1.4).
